### PR TITLE
Update issue templates (#3895)

### PR DESCRIPTION
**Pull request by jekkos**
Rebased @odiea work for the templates on master branch so we can use it in our repo.

### CODE REVIEW

**Review by odiea — APPROVED** [review]
Looks good to me.

File: .github/ISSUE_TEMPLATE/bug report.yml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        Bug reports indicate that something is not working as intended. 
+        Please include as much detail as possible and submit a separate bug report for each problem.
+        Do not include personal identifying information such as email addresses or encryption keys.           
+  - type: dropdown
+    id: ospos-version
+    attributes:
+      label: OpensourcePOS Version
+      description: What version of our software are you running?
+      options:
+        - opensourcepos 3.3.9
```
Comment (objecttothis) [inline .github/ISSUE_TEMPLATE/bug report.yml:21]: Does it make sense here to list versions back to 3.3.2?

**Comment by odiea** [comment]
My thoughts were if there are any old or new bugs introduced in these versions they should be dealt with. It is up to the user to update their versions to keep up with the current OSPOS versions.
I also would like to see the new CI4 version should start with 4.0.0 to show we are now on CI4.

**Comment by objecttothis** [comment]
> My thoughts were if there are any old or new bugs introduced in these versions they should be dealt with. It is up to the user to update their versions to keep up with the current OSPOS versions. I also would like to see the new CI4 version should start with 4.0.0 to show we are now on CI4.

@jekkos and I talked about this awhile back.  Even though there is a significant amount of work involved, @jekkos had a good point that the update doesn't by itself add any new features, only scaffolding for a lot of really cool features. For this reason, he argued to make it a minor version change 3.4.0 instead of 4.0.0.  Likely someone will want to redo the UI in the latest version of bootstrap and make it more responsive and that would likely warrant a major version change.  Also since we are planning introducing API's and 3rd party integrations, those would also be factors in a major version change.

**Review by objecttothis — APPROVED** [review]

**Pull request merged**